orbstation/code/mob/npc/bees.dm

```diff
@@ -0,0 +1,11 @@
+// Variant of the poisonous bees summoned by the "Anthophillia" holodeck simulation, which die after stinging once.
+/mob/living/simple_animal/hostile/bee/toxin/holo
+	name = "holographic bee"
+	desc = "Someone should probably turn that simulation off, huh?"
+
+/mob/living/simple_animal/hostile/bee/toxin/holo/AttackingTarget()
+	..()
+	death(gibbed = FALSE)
```

Comment:
Ignoring the voice in my head which is saying "this could be an element", it's absolutely not necessary for it to be one.